wallet/bindings/capacitorjs/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@iota/wallet-mobile",
+  "version": "2.0.0-alpha.0",
```

Comment:
Would this really be v2? I can't find v0 or v1 on npm

Reply:
I've use it to keep parity with the `nodejs` folder. But we could change it as you say ofc. My reasoning was the v1 was for Chrysalis, v2 for Stardust. Maybe we can upload to npm both versions with their respective version numbers?
Also related to this, we should use `2.0.0-rc.0` instead `2.0.0-alpha.0` to keep parity with other bindings?

Reply:
v1/v2 doesn't necessarily mean chrysalis/stardust, all projects have their own independent versioning. The wallet v1 is stardust for example. Are we planning on releasing this stuff for chrysalis as well?

Reply:
Something should technically be released as an alpha then evolve into a beta then RC

Reply:
great will change it, thanks

Reply:
About the Chrysalis version, was not planned but we could if you think